### PR TITLE
Release patch update 5.17.1

### PR DESCRIPTION
**Pull request by IvanCraft623**

### CODE REVIEW

File: changelogs/5.17.md [hunk 1]
```diff
@@ -23,3 +23,15 @@ Consider using the `mcpe-protocol` directive in `plugin.yml` as a constraint if
...
 - Bells always drops themselves even when using an incompatible tool.
+
+# 5.17.1
+Released 13th August 2024.
+
+## Documentation
+- Added a note about `BlockStateData::CURRENT_VERSION`.
+
+## Fixes
+- Fixed outdated `BedrockWorldData` version, this was preventing use newer worlds.
```
Comment (dktapps) [inline changelogs/5.17.md:35]: Anvil rotation too

Comment (dktapps) [reply changelogs/5.17.md:35]: never mind, looks like this branch doesn't have that change

Comment (IvanCraft623) [reply changelogs/5.17.md:35]: Oops, seems to it was merged just as I was writing the changelogs

Comment (dktapps) [reply changelogs/5.17.md:35]: Yeah I figured. The release process really needs to be rethought.

Comment (IvanCraft623) [reply changelogs/5.17.md:35]: Solved

**Pull request merged**